vizro-core/docs/pages/user_guides/card_button.md

```diff
@@ -0,0 +1,703 @@
+# How to use cards & buttons
```

Comment:
Nothing new, I've just renamed the file from `components.md` to `card_button.md`